src/nimble.nim

```diff
@@ -251,10 +251,14 @@ proc buildFromDir(
     if not existsDir(outputDir):
       createDir(outputDir)
 
+    let input = realDir / bin.changeFileExt("nim")
+    # `quoteShell` would be more robust than `\"` (and avoid quoting when
+    # un-necessary) but would require changing `extractBin`
+    let cmd = "\"$#\" $# --noNimblePath $# $# $# \"$#\"" %
+            [getNimBin(), pkgInfo.backend, nimblePkgVersion,
+             join(args, " "), outputOpt, input]
     try:
-      doCmd("\"" & getNimBin() & "\" $# --noNimblePath $# $# $# \"$#\"" %
-            [pkgInfo.backend, nimblePkgVersion, join(args, " "), outputOpt,
-             realDir / bin.changeFileExt("nim")])
+      doCmd(cmd, showCmd = true)
```

Comment:
This doesn't check for ``verbose`` unless I'm missing something.

Reply:
without --verbose, `display("Executing", cmd, priority = MediumPriority)` won't print anything because of `MediumPriority`
with --verbose, `display("Executing", cmd, priority = MediumPriority)` will print

as you can verify with (after rebuilding nimble after this PR):
```
nim c -o:nimble2 src/nimble.nim
./nimble2 install -y https://github.com/genotrance/nimgen
./nimble2 install -y -p:-u:release --verbose https://github.com/genotrance/nimgen
```

(or `./nimble2 install -y -p:-u:release --verbose https://github.com/genotrance/nimgen` to verify the other change I did)